ambry-store/src/main/java/com.github.ambry.store/DefaultCompactionPolicy.java

```diff
@@ -62,6 +64,9 @@ public CompactionDetails getCompactionDetails(long totalCapacity, long usedCapac
         if (bestCandidateToCompact != null) {
           details = new CompactionDetails(validDataSizeByLogSegment.getFirst(),
               bestCandidateToCompact.getSegmentsToCompact());
+          logger.info("Best candidate to compact {} ", bestCandidateToCompact);
+        } else {
+          logger.trace("No best candidate found ");
```

Comment:
minor: sorry to be picky, but there are lot of log messages that have a trailing space at the end (like this one). Could you please remove?